ReadAlongs/Studio-Web
# fix: make the privacy dialogue fully visible on smaller screens

With a maxHeight of 95vh, we now automatically get a scroll bar if needed, and the Opt out of/in to analytics button is always visible.

Also up the width from 50vw to 60vw, which again is nicer on screens smaller than large monitors.





### PR Goal? 

Make the Opt out of / in to analytics button visible on smaller screens, and add a scroll bar so the whole policy can be read on them.

### Fixes? 

Fixes #352

### Feedback sought? 

sanity checking, more or less just rubber stamping for this one

### Priority? 

medium-high

### Tests added? 

no

### How to test? 

click on Privacy and shrink your viewport to see the Opt in/out button is still usable and the full text can still be accessed.

### Confidence? 

high

### Version change? 

no
